walledeval/judge/string/__init__.py

```diff
@@ -0,0 +1,42 @@
+# walledeval/judge/string/__init__.py
+from pathlib import Path
+
+import yaml
+
+from walledeval.judge import Judge
+
+__all__ = ["StringMatchingJudge"]
+
+
+class StringMatchingJudge(Judge[None, bool]):
+
+    def __init__(self, name: str, must_match_all: list[str] = None, must_mismatch_all: list[str] = None,
+                 caseless: bool = False):
+        super().__init__(name)
+
+        self.must_match_all, self.must_mismatch_all, self.caseless = StringMatchingJudge.from_preset(name) if (
```

Comment:
nah this is too overengineered you don't need to do that 🤡 relax just take in the lists as is